Doc/faq/library.rst

```diff
@@ -780,8 +780,8 @@ socket to :meth:`select.select` to check if it's writable.
    The :mod:`asyncio` module provides a general purpose single-threaded and
    concurrent asynchronous library, which can be used for writing non-blocking
    network code.
-   The third-party `Twisted <https://twistedmatrix.com/trac/>`_ library is
-   a popular and feature-rich alternative.
+   The third-party `Twisted <https://twisted.org/>`_ library is a popular and
+   feature-rich alternative.
```

Comment:
It’s not very important here, but minimizing diffs help history/blame queries.

```suggestion
   The third-party `Twisted <https://twisted.org/>`_ library is
   a popular and feature-rich alternative.
```

Reply:
No problem. If the idea is to minimize diff, I'll squash & force-push your suggestion.

Reply:
Force pushes on github do not make a great experience for reviewers, and are not useful as Python PRs are always squash merge.  Normal commits are fine.  (See https://devguide.python.org/getting-started/pull-request-lifecycle/index.html for our little quirks.)

Reply:
Ok, thanks for the feedback